frontend/src/pages/Room/index.js

```diff
@@ -13,11 +13,14 @@ import QuestionBox from '../../components/ui/question/QuestionBox';
 import CollabEditor from '../../components/ui/collaboration/CollabEditor';
 import styles from './Room.module.css';
 
-import { URL_COMM_SVC_CONNECT } from '../../configs';
-import { URL_COLLAB_SVC } from '../../configs';
+import { URL_COMM_SVC_CONNECT, URL_COLLAB_SVC_CONNECT } from '../../configs';
 
-const comm_socket = io(URL_COMM_SVC_CONNECT);
-const collab_socket = io(URL_COLLAB_SVC);
+const comm_socket = io(URL_COMM_SVC_CONNECT, {
+  path: '/api/communication-service/socket',
```

Comment:
Please put this hardcoded path into config.js, thanks! Let's consolidate all the URL into the file. 😄